apps/www/_blog/2022-08-08-launch-week-5-hackathon

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Launch Week 5 Hackathon'
+description: Build to win $1500 - Friday 12th to Monday 21st August 2022
+author: ant_wilson
+image: lw5-hackathon/thumb.png
+thumb: lw5-hackathon/thumb.png
+tags:
+  - launch-week
+date: '2022-08-08'
+toc_depth: 3
+---
+
+# The Supabase Open Source Hackathon
+
+Now is the perfect time to start your new Open Source project!
+
+[Launch Week](https://supabase.com/launch-week) is about to start and we're running a hackathon in parallel for the entire duration.
+
+There's a $1500 GitHub sponsorship and [gold Supabase caps](https://store.supabase.com/products/gold-supacap) to be won!
+
+[Supabase-js v2.0](https://github.com/supabase/supabase-js/releases) and [Flutter SDK v1.0](https://supabase.com/blog/supabase-flutter-sdk-1-developer-preview) will both be announced during Launch Week, they are both available already and using them in your project will gain extra credit!
+
+<iframe
+  className="video-with-border w-full"
+  width="640"
+  height="385"
+  src="https://www.youtube-nocookie.com/embed/rI3Ik7GyYEw"
+  frameBorder="1"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowFullScreen
+></iframe>
+
+## Key Facts
+
+- You have 10 days to build a new **Open Source** project using Supabase.
+  - Starting 8:00am PT Friday 12th August 2022.
+- Build whatever you want - a project, app, tool, library, anything
+- Enter as an individual, or as a team of up to 5 people
+- Submission deadline is 11:59pm Sunday night PT 21st August 2022
+- There are 5 prize categories.
+  - Best overall project will win $1500 in GitHub sponsorships!
+  - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a Gold Supabase Cap!
+
+![Gold cap](/images/blog/lw5-hackathon/spool.jpg)
+
+## Details
+
+### Schedule
+
+- The Hackathon begins **NOW**! (8:00am PT Friday 12th August 2022)
+- Work on your project any time for the next 10 days
+- Submission deadline (11:59pm PT Sunday 21st August 2021)
+- Winners Announced (10pm PT Monday 22nd August 2022)
```

Comment:
Should we be so tight with the announcement? Do this date give you enough time to select the winners?

Reply:
yeah maybe we should just leave the date off to give us breathing room